app/helpers/homepage_helper.rb

```diff
@@ -0,0 +1,23 @@
+module HomepageHelper
+  def organisations_path
+    Plek.find("content-store") + "/content/government/organisations"
+  end
+
+  def organisations_json
+    JSON.parse(RestClient.get(organisations_path))
```

Comment:
I'd probably [use gds-api-adapters for this](https://github.com/alphagov/gds-api-adapters/blob/main/lib/gds_api/content_store.rb#L13-L17), then you don't need to worry about using Plek either.

```suggestion
    GdsApi.content_store.content_item("/organisations")
```

Reply:
I also wonder if there's a case for using the Search API here instead, since this means a single request to the homepage requires two requests to the content store (the ideal architecture is that you'd only need to make one call to the content store for each page. However, in this case the homepage is hard coded content anyway so it's a bit of a special case.

Reply:
This is going to play havoc with the intro to tech architecture session 😢

Reply:
An alternative to using the Search API would be to link the homepage content item with the `/organisations` content item, and then [use link expansion](https://github.com/alphagov/publishing-api/blob/master/docs/link-expansion.md#introduction) on the parts of the `details` you need from the homepage to avoid the second request to the content store.

Reply:
(Although I'm a bit confused why the organisations at the moment are in the `details` of `/government/organisations` content item and not in the `links`...)

Reply:
I think it's "government/organisations", but that works!